tests/unit/errors/silent-test.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var SilentError    = require('silent-error');
+var SilentErrorLib = require('../../../lib/errors/silent');
+var expect         = require('chai').expect;
+
+describe('SilentError', function() {
+  it('return silent-error and print a deprecation', function() {
+    expect(SilentErrorLib, 'returns silent-error').to.equal(SilentError);
```

Comment:
@stefanpenner This prints a deprecation when testing, but I'm completely forgetting how I can capture that to validate.